src/tests/integration/D3MCompound.t.sol

```diff
@@ -300,26 +306,28 @@ contract D3MCompoundTest is DSSTest {
         assertEq(vat.dai(address(d3mHub)), 0);
     }
 
-    function test_target_increase() public {
-        // Lower by 50%
-        uint256 targetBorrowRate = _setRelBorrowTarget(5000);
+    function test_borrow_apy() public {
+        // target 2% borrow apy, see top of D3MCompoundPlan for the formula explanation
+        uint256 targetBorrowRate = 7535450719; // ((2.00 / 100) + 1) ^ (1 / 365) - 1) / 7200) * 10^18
+
+        d3mCompoundPlan.file("barb", targetBorrowRate);
+        d3mHub.exec(ilk);
         assertEqInterest(getBorrowRate(), targetBorrowRate);
 
-        uint256 amountSuppliedInitial = cDai.balanceOfUnderlying(address(d3mCompoundPool));
+        cDai.borrow(1 * WAD);
 
-        // Raise by 25%
-        targetBorrowRate = _setRelBorrowTarget(12500);
-        assertEqInterest(getBorrowRate(), targetBorrowRate);
+        uint256 borrowBalanceBefore = cDai.borrowBalanceCurrent(address(this));
 
-        uint256 amountSupplied = cDai.balanceOfUnderlying(address(d3mCompoundPool));
-        assertTrue(amountSupplied > 0);
-        assertLt(amountSupplied, amountSuppliedInitial);
-        (uint256 ink, uint256 art) = vat.urns(ilk, address(d3mCompoundPool));
-        assertEqRounding(ink, amountSupplied);
-        assertEqRounding(art, amountSupplied);
+        // fast forward 1 year while accruing interest each day
+        for (uint256 i = 1; i <= 365; i++) {
+            hevm.roll(block.number + 7200);
+            cDai.accrueInterest();
```

Comment:
Why is it needed to be done every day?

Reply:
Not sure, but it seems to improve the accrual precision - when expecting 1.02 I'm getting 1.020146 instead of 1.019803 (when just doing `hevm.roll(block.number + 7200 * 365)`).
It is probably a more realistic case as accrual will be done on any borrow/repay.

Reply:
ok got it